app/products/calls/alerts.ts

```diff
@@ -23,12 +23,18 @@ import {isSystemAdmin} from '@utils/user';
 import type {LimitRestrictedInfo} from '@calls/observers';
 import type {IntlShape} from 'react-intl';
 
-// Only allow one recording alert per call.
+// Only unlock when:
+// - Joining a new call.
+// - A new recording has started.
+// - Host has changed to current user.
 let recordingAlertLock = false;
 
 // Only unlock if/when the user starts a recording.
 let recordingWillBePostedLock = true;
 
+// Only unlock when starting/stopping a recording.
+let recordingErrorLock = false;
```

Comment:
To keep with the pattern used in `recordingAlertLock` this should be set to true (meaning the recording error is not ready to show). In actuality it doesn't matter, since it's set to false before it's needed anyway, but at least the "default" state of true is made more clear if we default it to true, you know?